fhir-audit/src/main/java/com/ibm/fhir/audit/cadf/model/CadfReporterStep.java

```diff
@@ -72,8 +72,9 @@ public Date getReporterTime() {
          * it is contained within.
          * 
          * @param role    - The role the REPORTER performed on the CADF Event Record
-         *                (e.g., an "observer", "modifier" or "relay" role). @see
-         *                {@link CadfEvent#ReporterRole}
+         *                (e.g., an "observer", "modifier" or "relay" role). \
```

Comment:
is the `\` needed for line continuation or something?

Reply:
not intentional

Reply:
removed